### PR TITLE
Gui UserMan update (Part 4)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Issue #57.

- Line & Bar chart examples

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/bar.md [hunk 1]
```diff
@@ -4,4 +4,124 @@ Bar charts can be handy when you need to compare data point
...
+election in the US since 1852 (source
+[Wikipedia](https://en.wikipedia.org/wiki/List_of_United_States_presidential_elections_by_popular_vote_margin])).
+
+The Python code will look like this:
+```py
+percentages=[(1852,50.83), (1856,45.29), ..., (2016,46.09), (2020,51.31)]
+data = pd.DataFrame(percentages, columns= ['Year','%'])
+```
+
+A Pandas DataFrame is built from a list of tuples that hole the election year,
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bar.md:21]: hole => holds

File: docs/manuals/gui/viselements/charts/bar.md [hunk 2]
```diff
@@ -4,4 +4,124 @@ Bar charts can be handy when you need to compare data point
...
+  
+    === "HTML"
+
+        ```html
+        <taipy:chart type="bar" x="Year" y="%">{data}</taipy:chart>
+        ```
+
+All we did is set the _type_ property to "bar", and the following chart
+is displayed in the page (the blue color is not related to what party the
+elected president belongs to - it is just the default color that Plotly
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/charts/bar.md:42]: :-)

File: tools/assets/stylesheets/avaiga.css [hunk 3]
```diff
@@ -19,6 +19,9 @@ a.ext-link:after {
   height: 0.6em;
   margin: 0 3px 0 3px;
 }
+[data-md-color-scheme="slate"] a.ext-link:after {
+  filter: brightness(0) saturate(100%) invert(47%) sepia(40%) saturate(2289%) hue-rotate(340deg) brightness(101%) contrast(85%);;
```
Comment (FredLL-Avaiga) [inline tools/assets/stylesheets/avaiga.css:23]: waow

**Review by FredLL-Avaiga — APPROVED** [review]
+1

File: docs/manuals/gui/viselements/charts/bar.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/bar.md:21]: Thx!

File: tools/assets/stylesheets/avaiga.css (hunk 3 above)
Comment (FabienLelaquais) [reply tools/assets/stylesheets/avaiga.css:23]: Yeah well...

**Pull request merged**